rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,338 @@
+from typing import Optional
```

Comment:
Why not place this inside `tests/unit/accounting`? This is not a decoder test but an accounting test.

Reply:
Yes, makes sense